test/selenium/util/Config.scala

```diff
@@ -10,6 +10,8 @@ object Config {
 
   private val conf = ConfigFactory.load()
 
+  val guDomain = conf.getString("guDomain")
```

Comment:
I should probably change https://github.com/guardian/support-frontend/blob/master/app/controllers/TestUsersManagement.scala#L18 to use this new property.

No real need to shorten this to `gu`, `guardianDomain` would probably be clearer for new people.

Reply:
All the comments are addressed here: f72a26799fb356af300fbd8facfb12976d8d730f . Please, could you double check that the wiring part is correct?